test/test-notification-model-updates.js

```diff
@@ -0,0 +1,382 @@
+var ServiceManager = require('../index').ServiceManager;
+var async = require('async');
+var meshServer = require('../index').meshServer;
+var os = require('os');
+var test = require('tap').test;
+
+test('Test notifications', function(t) {
+  var app = meshServer(new ServiceManager());
```

Comment:
This seems like a good argument for making the `serviceManager` argument optional.